test/integration/dialects/abstract/connection-manager.test.js

```diff
@@ -125,10 +142,15 @@ describe(Support.getTestDialectTeaser('Connection Manager'), () => {
     };
 
     await connectionManager.getConnection(queryOptions);
-    chai.expect(deprecationStub).to.have.been.calledOnce;
+    chai.expect(stub).to.have.been.calledOnce;
+    chai
+      .expect(stub.getCalls()[0].args[0])
+      .to.contain(
+        'This database engine version is not supported, please update your database server.'
```

Comment:
I changed this to this format now. The rest of the file is just formatting.

Instead of checking that the deprecation function was called, I'm checking one level further down where deprecate is calling process.emitWarning.